shared-worker/index.html

```diff
@@ -0,0 +1,12 @@
+<script src="../firebase/firebase-app.js"> </script>
+<script src="../firebase/firebase-firestore.js"> </script>
+<script src="main.js" async></script>
+<h1>
+    Open dev tools to view tab state
```

Comment:
"to view tab state" isn't very clear.  I'd just change it to "to view script output" or come up with something else that's more explanatory of what's going on.